### DISCUSSION TITLE
BUG Arcade Shortcut not working in tadpole versions - FIX Workaround

### DISCUSSION THREAD

**Discussion by bn40400** [EricGoldsteinNz/tadpole#49] [Ideas]
***Original Discussion/ Question***

Hello, I've used the latest tadpole tool version 0.4.2, and I've tried to change my 4 favorite slots under the Mame (Arcade) section and none of the slots appear to work. I'll select them, get a slight flicker and loading screen, then immediately goes back to the home screen. I was able to successfully change the Famicom shortcuts with no issue, they work well. I just can't get the arcade ones to work. Any help and suggestions would be appreciated - thanks! 

***Edit***

I figured it out, arcade shortcuts in the Resources/xfgle.hgp are created incorrectly. They point to the .zfb files in the ARCADE folder rather than the corresponding .zip file in the ARCADE/bin folder. I was able to manually edit the invalid entries using notepad to the correct ones and the shortcuts now work. Hope this helps.

**Comment by EricGoldsteinNz** [comment]
This should be fixed in the v.5. release. Can you try again and let us know how you get on please?

**Reply by bn40400** [reply]
Having tested changing the shortcuts for only the arcade, I've only attempted to change one. In doing so, it completely wiped out and cleared the other arcade shortcuts I've created in v0.4.2. I've tried to recreate the shortcuts, and the v0.5.0 Tadpole will not accept the shortcuts (1-4). Upon reloading the ROM list, the shortcuts do not register at all. Currently, I'm using version 1.6 of the Data Frog SF2000 OS. For the time being, I've reverted back to Tadpole 0.4.2 and the work around I've discovered for the time being that is already mentioned previously in this post - until more version updates are available. Thank you all for your contributions, dedication and hard work (aka blood, sweat & tears). Looking forward to more future updated versions. I'm not afraid to test out a work in progress, I create backup versions of my Data Frog as well so if anything goes wrong, I can easily flash my SD and fall back on my backups. Keep them updates coming, cheers!

**Reply by bn40400** [reply]
Fixed in 1.0! Awesome job and thank you!!!

**Comment by EricGoldsteinNz** [comment]
Awesome! Thank you for your patience on this issue.
